changelog/@unreleased/pr-4259.v2.yml

```diff
@@ -0,0 +1,6 @@
+type: improvement
+feature:
+  description: KVS now has a getAsync endpoint which returns ListenableFuture. CassandraKVS uses CQL client for
```

Comment:
KVS -> Key/Value service or `KeyValueService`. It's not really an endpoint in the traditional sense. It's also quite impl heavy, for consumers (as this is a brand new feature) they mostly want to know a high level overview, consider using the below:

> Key/value services can now support asynchronous get operations. This has been implemented for the Cassandra key value service.